deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -618,3 +618,65 @@ cephObjectStores:
       #     - objectstore.example.com
       #   secretName: ceph-objectstore-tls
       # ingressClassName: nginx
+
+# cephECBlockPools are disabled by default, please remove the comments and set desired values to enable it
+#cephECBlockPools:
+#  - name: ceph-ec-blockpool
+#    # see https://github.com/rook/rook/blob/master/Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md#spec for available configuration
+#    spec:
+#      failureDomain: host
+#      replicated:
```

Comment:
This should have [erasure coded ](https://github.com/rook/rook/blob/master/Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md#erasure-coded)settings rather than replicated settings.